### ISSUE TITLE
Any suggestions on resource mapping?

### ISSUE THREAD

**Issue by astewart-twist** [Illumina/Pisces#25]
Just curious if there are any recommended minimum cpu/memory requirements for optimal run time, perhaps as a function of data size ?

**Comment by tamsen** [comment]
Hi!

Yes, that's a really good question. We did some benchmarking with the Pisces 5.2.7 release used for the paper, using the exact command lines for that analysis. Results:

"An average runtime for the
Pisces Variant Caller on a 470 MB BAM (8 million reads) is 85 seconds.
Runtime for a 2 GB BAM (60 million reads) is about 4 minutes, and for
a 65 GB BAM (900 million reads), about 40 minutes (35 seconds per GB
processed). These examples were run with 20 threads, on a Linux Cluster
node with Intel(R) Xeon(R) CPU E5-2670 @ 2.60GHz processors. "

We also have a spread sheet somewhere for memory use & timing by bam size, but it ended up being highly dependent on algorithms elected (min freq, RMxN, vcf or gvcf), regions (lots of repeats, variant hotspots), noise levels, MNV lengths,  etc, that we had to tailor the benchmarking each time to people's anticipated use cases.

I can look around for that spread sheet, and I will let you know if I find it. My general recommendation, is just bench mark for your self using a few typical samples.  As depth increases Pisces becomes more efficient, not less, so a linear extrapolation will give you a safe upper bound.

I hope that helps!
